tests/test_matrix_utils_relay.py

```
@@ -384,7 +384,7 @@ async def test_matrix_relay_importerror_fallback(
     real_import = builtins.__import__
 
     def fake_import(name, *args, **kwargs):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `fake_import`

(ANN202)